2-extra/4-eligible-students.js

```diff
@@ -14,7 +14,19 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+
+
+
+function eligibleStudents(attendances) {
+  students =[]
+  let result = attendances.filter(student => student.attendance >= 8);
+  result.forEach((studentName) => {
+    students.push(studentName.name)
```

Comment:
The logic is solid but in here you are using a forEach loop to go for each of the StudentName and then add them into students array. Is there another type of loop that is more suitable for this use case?